docs/how-to/hip_runtime_api/initialization.rst

```diff
@@ -0,0 +1,124 @@
+.. meta::
+   :description: Initialization.
+   :keywords: AMD, ROCm, HIP, initialization
+
+********************************************************************************
+Initialization
+********************************************************************************
+
+Initialization involves setting up the environment and resources needed for GPU
+computation. The following steps are covered with the initialization:
```

Comment:
```suggestion
The initialization involves setting up the environment and resources needed for
using GPUs. The following steps are covered with the initialization:
```

Also this should mention, that initialization is done automatically by HIP, and does not *need* to be done manually, but can be done manually for more control (timing on when HIP is initialized, what GPU to use)

Reply:
We are mentioning this later.